### DISCUSSION TITLE
How to hide a blog post without deleting the file

### DISCUSSION THREAD

**Discussion by ma-sadeghi** [alshedivat/al-folio#1251] [Q&A] [answered]
I just want to hide a blog post from the website without deleting it. I already found a workaround: add a leading underscore to the filename, but I was thinking maybe there is a directive like `publish: false` that I can modify. Thanks!

**Comment by ma-sadeghi** [comment, marked as answer]
Ok, I just found [this](https://stackoverflow.com/a/27799716/3438239) on SO. I just need to add `published: false` directive to the blog post header!

**Comment by george-gca** [comment]
You could also move it to `_drafts` directory. To visualize your draft posts, simply call jekyll with the `--drafts` flag, as in `bundle exec jekyll serve --lsi --drafts`. This way you can visualize locally a post while it's still incomplete, but avoid displaying it on the deployed site.
